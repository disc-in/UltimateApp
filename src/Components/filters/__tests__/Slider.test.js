import React from 'react';
import { render } from '@testing-library/react-native';

import Slider from '../Slider';

describe('<Slider />', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<Slider value={15} />);
    expect(toJSON()).toMatchSnapshot();
  });
});
