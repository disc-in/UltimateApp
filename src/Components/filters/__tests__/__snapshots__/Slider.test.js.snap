// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slider /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "paddingHorizontal": 10,
    }
  }
>
  <Text
    style={
      Object {
        "flexGrow": 1,
        "textAlign": "center",
      }
    }
  />
  <View
    allowTouchTrack={false}
    animationType="timing"
    onLayout={[Function]}
    orientation="horizontal"
    step={0}
    style={
      Object {
        "flexGrow": 10,
        "height": 40,
        "justifyContent": "center",
        "marginHorizontal": 10,
      }
    }
    theme={
      Object {
        "colors": Object {
          "black": "#242424",
          "disabled": "hsl(208, 8%, 90%)",
          "divider": "#bcbbc1",
          "error": "#ff190c",
          "grey0": "#393e42",
          "grey1": "#43484d",
          "grey2": "#5e6977",
          "grey3": "#86939e",
          "grey4": "#bdc6cf",
          "grey5": "#e1e8ee",
          "greyOutline": "#bbb",
          "platform": Object {
            "android": Object {
              "error": "#f44336",
              "grey": "rgba(0, 0, 0, 0.54)",
              "primary": "#2196f3",
              "searchBg": "#dcdce1",
              "secondary": "#9C27B0",
              "success": "#4caf50",
              "warning": "#ffeb3b",
            },
            "default": Object {
              "error": "#ff3b30",
              "grey": "#7d7d7d",
              "primary": "#007aff",
              "searchBg": "#dcdce1",
              "secondary": "#5856d6",
              "success": "#4cd964",
              "warning": "#ffcc00",
            },
            "ios": Object {
              "error": "#ff3b30",
              "grey": "#7d7d7d",
              "primary": "#007aff",
              "searchBg": "#dcdce1",
              "secondary": "#5856d6",
              "success": "#4cd964",
              "warning": "#ffcc00",
            },
            "web": Object {
              "error": "#ff190c",
              "grey": "#393e42",
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#ca71eb",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          },
          "primary": "#2089dc",
          "searchBg": "#303337",
          "secondary": "#ca71eb",
          "success": "#52c41a",
          "warning": "#faad14",
          "white": "#ffffff",
        },
      }
    }
    thumbTouchSize={
      Object {
        "height": 40,
        "width": 40,
      }
    }
    value={15}
  >
    <View
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "#b3b3b3",
          "borderRadius": 2,
          "height": 2,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "#2a9e91",
          "borderRadius": 2,
          "height": 0,
          "marginTop": -0,
          "position": "absolute",
          "width": 0,
        }
      }
    />
    <View
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "#2a9e91",
          "borderRadius": 20,
          "height": 20,
          "position": "absolute",
          "transform": Array [
            Object {
              "translateX": 0,
            },
          ],
          "width": 20,
        }
      }
      testID="sliderThumb"
    />
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "flexGrow": 1,
        "textAlign": "center",
      }
    }
  />
</View>
`;
